CONTRIBUTING.md

```diff
@@ -558,17 +562,17 @@ arguments and return values.
 
 ### Code Review
 
-After you've submitted a pull request, you should expect to hear at least a comment
-within a couple of days.
-We may suggest some changes or improvements or alternatives.
+After you've submitted a pull request, you should expect to hear at least a
+comment within a couple of days. We may suggest some changes, improvements or
+alternative implementation details.
 
-Some things that will increase the chance that your pull request is accepted quickly:
+To increase the chances of getting your pull request accepted quickly, try to:
```

Comment:
How about ordering the list like this:

- Write a good and detailed description of what the PR does

**Related to PR itself** (This is only my comments in this PR)

- Use readable code, as it is better than clever code (even with comments).
- Follow the PEP8 style guide for code and the numpy guide for documentation

**Related to coding style** (This is only my comments in this PR)

- Write documentation for your code (docstrings) and leave comments explaining the reason behind non-obvious 
things.
- Write tests for the code you wrote/modified.
- Include an example of new features in the gallery or tutorials.

**Related to PyGMT** (This is only my comments in this PR)

Reply:
Use nested lists in d0e8fe7.